collection-scripts/mce-gather.sh

```diff
@@ -0,0 +1,146 @@
+    oc adm inspect hiveconfigs.hive.openshift.io --all-namespaces  --dest-dir=must-gather
+
+    oc adm inspect clusterserviceversions.operators.coreos.com --all-namespaces  --dest-dir=must-gather
+    oc adm inspect subscriptions.operators.coreos.com      --all-namespaces  --dest-dir=must-gather
+    oc adm inspect installplans.operators.coreos.com --all-namespaces  --dest-dir=must-gather
+    oc adm inspect operatorgroups.operators.coreos.com --all-namespaces  --dest-dir=must-gather
+
+    oc adm inspect baremetalhosts.metal3.io --all-namespaces  --dest-dir=must-gather
+
+    oc adm inspect placementdecisions.cluster.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect placements.cluster.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect clusterdeployments.hive.openshift.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect syncsets.hive.openshift.io --all-namespaces --dest-dir=must-gather
+    oc adm inspect clusterimagesets.hive.openshift.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect machinesets.machine.openshift.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect clustercurators.cluster.open-cluster-management.io --all-namespaces --dest-dir=must-gather
+
+    oc adm inspect  managedclusterviews.view.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  managedclusteractions.action.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  manifestworks.work.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  managedclusters.cluster.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  managedclusterinfos.internal.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  clustermanagers.operator.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  managedserviceaccounts.authentication.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+
+
+    oc adm inspect validatingwebhookconfigurations.admissionregistration.k8s.io --all-namespaces --dest-dir=must-gather
+    oc adm inspect mutatingwebhookconfigurations.admissionregistration.k8s.io --all-namespaces --dest-dir=must-gather
+
+    oc adm inspect  discoveredclusters.discovery.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+    oc adm inspect  discoveryconfigs.discovery.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
+
+    oc adm inspect managedclusteraddons.addon.open-cluster-management.io  --all-namespaces  --dest-dir=must-gather
+
+    oc adm inspect ns/openshift-monitoring  --dest-dir=must-gather
+
+    # OpenShift console plug-in enablement
+    oc adm inspect consoles.operator.openshift.io --dest-dir=must-gather
+}
+
+check_if_hub
+check_if_spoke
+
+if $HUB_CLUSTER;
+then
+  echo "Start to gather information for hub"
```

Comment:
Update log message to be more specific and different from the other similar log.
```suggestion
  echo "Start to gather MCE information for hub"
```